platform/solr/solr-password/src/main/java/ddf/platform/solr/security/SolrPasswordUpdate.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.platform.solr.security;
+
+public interface SolrPasswordUpdate {
+
+  /** Begin the passsword update process */
+  void start();
```

Comment:
should this be named something more descriptive like `generateSolrPassword` or something like that.

Reply:
Sure. How about `udpateSolrPassword`?